src/submissions/authors/containers/AuthorSubmissionPage.jsx

```diff
@@ -35,7 +35,11 @@ class AuthorSubmissionPage extends Component {
           </Row>
           <Row>
             <Col>
-              <AuthorSubmission error={error} onSubmit={this.onSubmit} />
+              <AuthorSubmission
+                error={error}
+                onSubmit={this.onSubmit}
+                initialFormData={query}
```

Comment:
this is too vulnerable, users can pass anything via url params to form data.

So:
```typescript
const { bai } = query;
const initialFormData = { bai };
```